### ISSUE TITLE
Increase accessibility by providing visual change in interface elements on keyboard focus

### ISSUE THREAD

**Issue by dovematrix** [mpetroff/pannellum#510]
Hi Matt.  Can the user interface elements that can receive keyboard focus, be highlighted on focus? 

I noticed the controls are not highlighted on focus when using the tab key on my keyboard.

To provide accessibility with keyboard navigation it is important that the focused component can be visually identified. If the focused component is not somehow highlighted, the user cannot be sure which component has focus.

Thanks,
Russell

**Comment by mpetroff** [comment]
The controls don't have a tab index, so they don't receive focus (as far as I know). Since they don't receive focus, they aren't highlighted.

**Comment by dovematrix** [comment]
Is it possible to give them a tab index?

**Comment by mpetroff** [comment]
Looking at various web maps, e.g. Google Maps, Leaflet, Mapbox GL, etc., most don't use tab indicies for the zoom controls, or the ones that do don't show a visual indication, so Pannellum is by no means the exception in this regard. I experimented with adding a visual indicator to the buttons, but I find it quite distracting. I could add tab indicies without a highlight?

**Comment by adamjohnson** [comment]
> One of the most significant things you can do to improve the accessibility of your work is to create highly visible focus indicators.

[-Derek Featherstone](https://twitter.com/feather/status/1088702913487106049)

There's also an article from The Paciello Group about creating [accessible digital map experiences.](https://developer.paciellogroup.com/blog/2020/04/accessible-digital-map-experiences/). Not a lot online about accessible maps, but that article has a few tips and links to a few decent resources. Pannellum does pretty well here with arrow and `+`/`-` keys.
